exec/cnex/cnex.c

```diff
@@ -1371,10 +1388,19 @@ void exec_CALLP(TExecutor *self)
     _IDEC_glbflags = 0;
     unsigned int start_ip = self->ip;
     self->ip++;
+     g_executor->diagnostics.total_predef_calls++;
```

Comment:
This line does not appear to be indented correctly

Reply:
Fixed.